tests/test_cli.py

```diff
@@ -112,6 +112,21 @@ def test_cli_script(disable_cli):
     assert "foo.py" in args
 
 
+@pytest.mark.cli
+def test_cli_scriptargs(disable_cli):
+    args, _ = _cli_impl(
+        exe="AnsysWBU.exe",
+        version=241,
+        input_script="foo.py",
+        script_args="arg1, arg2, arg3",
```

Comment:
remove the spaces in the string, that's more conventional for this kind of thing